### ISSUE TITLE
 Please rebuild and republish the charm to the charmstore

### ISSUE THREAD

**Issue by axinojolais** [omnivector-solutions/layer-sentry#1]
Current charmstore version is broken because it doesn't have the latest redis interface layer, and does :
```
            for unit in relation.units:
                host = unit.received_raw['host']
                port = unit.received_raw['port']
```
where it should be :
```
            for unit in relation.units:
                host = unit.received['host']
                port = unit.received['port']
```

Thanks

**Comment by jamesbeedy** [comment]
closed by #1

**Comment by jamesbeedy** [comment]
@axinojolais thanks for getting this filed. Try out `juju deploy cs:~omnivector/bundle/sentry-decoupled-2` for the latest bits.
